shared/src/commonMain/kotlin/org/hyperskill/app/step_quiz/injection/StepQuizComponentImpl.kt

```diff
@@ -41,27 +30,27 @@ class StepQuizComponentImpl(
     override val stepQuizUserPermissionRequestTextMapper: StepQuizUserPermissionRequestTextMapper
         get() = StepQuizUserPermissionRequestTextMapper(appGraph.commonComponent.resourceProvider)
 
-    override val stepQuizInteractor: StepQuizInteractor
-        get() = StepQuizInteractor(
+    private val attemptRemoteDataSource: AttemptRemoteDataSource = AttemptRemoteDataSourceImpl(
+        appGraph.networkComponent.authorizedHttpClient
+    )
+    private val attemptRepository: AttemptRepository =
+        AttemptRepositoryImpl(attemptRemoteDataSource)
+
+    override val stepQuizInteractor: StepQuizInteractor =
+        StepQuizInteractor(
             attemptRepository,
             appGraph.submissionDataComponent.submissionRepository
         )
 
     override val stepQuizFeature: Feature<StepQuizFeature.State, StepQuizFeature.Message, StepQuizFeature.Action>
-        get() {
-            val stepQuizReducer = StepQuizReducer(stepRoute)
-            val stepQuizActionDispatcher = StepQuizActionDispatcher(
-                ActionDispatcherOptions(),
-                stepQuizInteractor,
-                stepQuizReplyValidator,
-                appGraph.buildProfileDataComponent().profileInteractor,
-                appGraph.buildNotificationComponent().notificationInteractor,
-                appGraph.analyticComponent.analyticInteractor,
-                appGraph.sentryComponent.sentryInteractor,
-                appGraph.commonComponent.resourceProvider
-            )
-
-            return ReduxFeature(StepQuizFeature.State.Idle, stepQuizReducer)
-                .wrapWithActionDispatcher(stepQuizActionDispatcher)
-        }
+        get()  = StepQuizFeatureBuilder.build(
```

Comment:
space `get()  =` -> `get() =`

Reply:
@ivan-magda  Can we check such cases with ktlint?

Reply:
Not sure, maybe after [ALTAPPS-490](https://vyahhi.myjetbrains.com/youtrack/issue/ALTAPPS-490)